live_data_subscriber/.envvars

```diff
@@ -3,12 +3,12 @@
 #     set -a ; source .envvars ; set +a
 #
 # in order to set up the API client and MQTT connections
-
 MQTT_USERNAME=""
 MQTT_PASSWORD=""
 MQTT_DOMAIN_NAME=""
 API_CLIENT_USERNAME=""
 API_CLIENT_PASSWORD=""
-API_CLIENT_DOMAIN_NAME="http://localhost:8000"
-API_CLIENT_WEBSOCKET_DOMAIN_NAME="ws://localhost:8000/external-ws"
+API_CLIENT_DOMAIN_NAME=""
```

Comment:
Is there a specific reason why we're not using something like [python-dotenv](https://github.com/theskumar/python-dotenv) here, rather than running `set -a ; source .envvars ; set +a`?

Reply:
Its been there before this PR, I would actually ask @spyrostz of having this as a preference.

Reply:
Because this is supposed to be a convenient function for local testing, not a way to set configuration in real operation of the live_data subscriber. These env variables are all supposed to be handled by the K8 config map in the deployed live_data_subscriber, therefore the .env file would not be useful. Therefore I would be very hesitant to add a dependency + Python code that reads .env files without any need for such code in production, especially since it is an one-liner to read the file locally.